server/container_create_linux_test.go

```diff
@@ -71,3 +72,35 @@ func TestAddOCIBindsForSys(t *testing.T) {
 		t.Error("there is not a single /sys bind mount")
 	}
 }
+
+func TestRdtClassFromAnnotations(t *testing.T) {
```

Comment:
nit: we should be moving more tests to ginkgo, rather than testing. can these be written using ginkgo

Reply:
These tests were dropped (moved over to goresctrl codebase) when moving the annotation parsing logic there